onnx/helper.py

```diff
@@ -1533,3 +1533,34 @@ def get_all_tensor_dtypes() -> KeysView[int]:
     :return: all tensor types from TensorProto
     """
     return mapping.TENSOR_TYPE_MAP.keys()
+
+
+attribute_proto_type_to_str = {
+    AttributeProto.FLOAT: "float",
+    AttributeProto.FLOATS: "floats",
+    AttributeProto.GRAPH: "graph",
+    AttributeProto.GRAPHS: "graphs",
+    AttributeProto.INT: "int",
+    AttributeProto.INTS: "ints",
+    AttributeProto.SPARSE_TENSOR: "sparse_tensor",
+    AttributeProto.SPARSE_TENSORS: "sparse_tensors",
+    AttributeProto.STRING: "string",
+    AttributeProto.STRINGS: "strings",
+    AttributeProto.TENSOR: "tensor",
+    AttributeProto.TENSORS: "tensors",
+    AttributeProto.TYPE_PROTO: "proto",
+    AttributeProto.TYPE_PROTOS: "protos",
+}
```

Comment:
Thank you for the enhancement!

This map (attribute_proto_type_to_str) seems more proper to exist in https://github.com/onnx/onnx/blob/main/onnx/mapping.py?

Reply:
We wouldn't want to expose it unless there is a need? I think keeping it private for now makes sense

Reply:
Yes, we should set `attribute_proto_type_to_str` as private function at this moment to prevent someone starts to rely on this map directly, although in long term ONNX will eventually make mapping.py as _mapping.py: https://github.com/onnx/onnx/issues/4554.

Reply:
I see. In this case maybe move the dictionary into _mapping after it becomes private?